test/feed-worker.test.js

```diff
@@ -0,0 +1,48 @@
+const feedQueue = require('../src/feed-queue');
+const feedWorker = require('../src/feed-worker');
+
+// Bull will hold the redis connection open unless we shutdown the queue,
+// which will block jest from completing.  After all tests are done, close it.
+afterAll(() => feedQueue.close());
+
+describe('testing the worker callback function', () => {
+  feedWorker.start();
+
+  const job1 = { data: 'https://c3ho.blogspot.com/feeds/posts/default/-/open-source?alt=rss' };
+  const job2 = { data: 'c3ho.blogspot.com/feeds/posts/default/-/open-source?alt=rss' };
+  const job3 = { data: 'https://c3ho.blogspot.com/feeds/posts/default/-/open-source' };
+  // const job4 = { data: 'c3ho.blogspot.com/feeds/posts/default/-/open-source' };
```

Comment:
Is there a reason why these are still here? if they're not going to be used, maybe remove them?

Reply:
They are going to be used. It's just that even the first three tests are not passing right now, so they are temporarily commented out.